lewis/adapters/epics.py

```diff
@@ -205,12 +206,17 @@ def start_server(self):
 
             The server does not process requests unless :meth:`handle` is called regularly.
         """
-        self._server = SimpleServer()
-        self._server.createPV(prefix=self._options.prefix,
-                              pvdb={k: v.config for k, v in self.pvs.items()})
-        self._driver = PropertyExposingDriver(target=self, pv_dict=self.pvs)
+        if self._server is None:
+            self._server = SimpleServer()
+            self._server.createPV(prefix=self._options.prefix,
+                                  pvdb={k: v.config for k, v in self.pvs.items()})
+            self._driver = PropertyExposingDriver(target=self, pv_dict=self.pvs)
+
+            self._last_update = datetime.now()
 
-        self._last_update = datetime.now()
+    def stop_server(self):
+        self._server = None
+        self._driver = None
```

Comment:
I haven't tried in connection with #169, but in principle this uses the same mechanism as PV value updates (and on the same poll interval). But now that you mention it we should probably have one forced poll after instantiating the driver so that everything is updated. Since all that is happening in one cycle I don't think there will be a problem.